src/Orc.Controls.Examples/ViewModels/DatePickerViewModel.cs

```diff
@@ -21,6 +25,36 @@ public DatePickerViewModel()
 
         #region Properties
         public DateTime DateValue { get; set; }
+        public FastObservableCollection<object> AvailableFormats { get; private set; }
+        public object SelectedFormat { get; set; }
+        #endregion
+
+        #region Methods
+        protected override async Task InitializeAsync()
+        {
+            await base.InitializeAsync();
+
+            AvailableFormats = new FastObservableCollection<object>();
```

Comment:
Best is to instantiate collections inside the ctor. Sometimes you have commands (someone might add it) and it breaks with a NullReference when people expect this property to have a value.